evm-contracts/test/v0.6/PrepaidAggregator.test.ts

```diff
@@ -1679,4 +1681,149 @@ describe('PrepaidAggregator', () => {
       matchers.bigNum(originalBalance.add(deposit), newBalance)
     })
   })
+
+  describe('#startNewRound', () => {
+    beforeEach(async () => {
+      await aggregator
+        .connect(personas.Carol)
+        .addOracle(personas.Neil.address, personas.Neil.address, 1, 1, 0)
+
+      await aggregator.connect(personas.Neil).updateAnswer(nextRound, answer)
+      nextRound = nextRound + 1
+
+      await aggregator.setAuthorization(personas.Carol.address, true)
+
+      await aggregator.connect(personas.Carol)
```

Comment:
What does this do?

Reply:
Nothing, I thought that `connect`modified the aggregator, but it returns a modified instance. Understanding that now, I'll remove this line. Good catch.